dataprofiler/profilers/profile_builder.py

```diff
@@ -1393,7 +1388,7 @@ def tqdm(l):
         clean_sampled_dict = {}
         multi_process_dict = {}
         single_process_list = set()
-        if not sample_size: sample_size = len(data)
+        
         if sample_size < len(data):
```

Comment:
if you delete the above line, does that work for this if statement when sample_size is None?

Reply:
sample_size cannot be `none`

Reply:
If a user doesn't set sample_size in the func `update_profile`, it is autoset within that function before it is passed to this one.

Reply:
kk